### PR TITLE
[docs]: Fix documentation for Link Component

### PR DESCRIPTION
**Pull request by AsifQamar**
@RudraPratik30 here the link.md changes

### CODE REVIEW

File: docs/docs/widgets/link.md [hunk 1]
```diff
@@ -47,15 +46,15 @@ Following actions of link component can be controlled using the component specif
...
 
 There are currently no exposed variables for the component.
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## General
 ### Tooltip
```
Comment (RudraPratik30) [inline docs/docs/widgets/link.md:60]: Hey @AsifQamar here on line no. 62 you need to change the widget to the component.

File: docs/docs/widgets/link.md [hunk 2]
```diff
@@ -86,10 +85,10 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 | Text color |  You can change the background color of the text by entering the Hex color code or choosing a color of your choice from the color picker. |
 | Text size | By default, the text size is set to 14. You can enter any value from 1-100 to set custom text size. |
 | Underline | You can change the underline of the text in the following ways: **on-hover (default), never, always** |
-| Visibility | Toggle on or off to control the visibility of the widget. You can programmatically change its value by clicking on the `Fx` button next to it. If `{{false}}` the widget will not visible after the app is deployed. By default, it's set to `{{true}}`. |
-
+| Visibility | Toggle on or off to control the visibility of the widget. You can programmatically change its value by clicking on the **fx** button next to it. If `{{false}}` the component will not visible after the app is deployed. By default, it's set to `{{true}}`. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/link.md:88]: Here on line no. 88 as well, change the widget to component

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @AsifQamar, please make the mentioned changes in both versions, also please add full stops wherever needed.

**Review by RudraPratik30 — APPROVED** [review]
Hey @AsifQamar, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.
You can expect your PR to be merged by the end of this week, thanks!

**Review by johnsoncherian — APPROVED** [review]

**Comment by PriteshKiri** [comment]
#10969

**Pull request merged**